src/main/java/seedu/address/model/person/assignment/Grade.java

```diff
@@ -48,6 +51,50 @@ public Grade copyGrade() {
         }
     }
 
+    /**
+     * Checks if the grade being added is valid.
+     *
+     * @param test Grade to be added.
+     * @param maxGrade Maximum Grade.
+     * @return Whether the grade being added is valid.
+     */
+    public static boolean isValidGrade(String test, String maxGrade) {
+        int testInteger;
+        int maxInteger;
```

Comment:
Will the grades have decimal places? If yes, then "int" might not be the most suitable data type to use
(ps, i forget how source grading works haha)

Reply:
Idt have decimal, that's why using int haha